src/components/RegisterForm.vue

```diff
@@ -0,0 +1,276 @@
+				{{ v$.email.required.$message.replace("Value", "Email") }}
+			</small>
+		</div>
+		<div class="field mb-4">
+			<div class="p-float-label">
+				<Password
+					id="password"
+					v-model="v$.password.$model"
+					:class="{ 'p-invalid': v$.password.$invalid && submitted }"
+					toggleMask>
+					<template #header>
+						<h6>Pick a password</h6>
+					</template>
+					<template #footer="sp">
+						{{ sp.level }}
+						<Divider />
+						<p class="mt-2">Suggestions</p>
+						<ul class="pl-2 ml-2 mt-0" style="line-height: 1.5">
+							<li>At least one lowercase</li>
+							<li>At least one uppercase</li>
+							<li>At least one numeric</li>
+							<li>Minimum 8 characters</li>
+						</ul>
+					</template>
+				</Password>
+				<label for="password" :class="{ 'p-error': v$.password.$invalid && submitted }">Password*</label>
+			</div>
+			<small v-if="(v$.password.$invalid && submitted) || v$.password.$pending.$response" class="p-error">
+				{{ v$.password.required.$message.replace("Value", "Password") }}
+			</small>
+		</div>
+		<div class="field mb-4">
+			<div class="p-float-label">
+				<InputText
+					id="street"
+					v-model="v$.address.street.$model"
+					:class="{ 'p-invalid': v$.address.street.$invalid && submitted }" />
+				<label for="street">Street*</label>
+			</div>
+			<span v-if="v$.address.street.$error && submitted">
+				<span id="street-error" v-for="(error, index) of v$.address.street.$errors" :key="index">
+					<small class="p-error">{{ error.$message }}</small>
+				</span>
+			</span>
+			<small
+				v-else-if="(v$.address.street.$invalid && submitted) || v$.address.street.$pending.$response"
+				class="p-error">
+				{{ v$.address.street.required.$message.replace("Value", "Street") }}
+			</small>
+		</div>
+		<div class="field mb-4 flex justify-content-between">
+			<div class="p-float-label w-7">
+				<div class="p-float-label w-12 pr-1">
+					<InputNumber
+						id="houseNumber"
+						v-model="v$.address.houseNumber.$model"
+						:class="{ 'p-invalid': v$.address.houseNumber.$invalid && submitted }" />
+					<label for="houseNumber">House number*</label>
+				</div>
+				<span v-if="v$.address.houseNumber.$error && submitted">
+					<span id="houseNumber-error" v-for="(error, index) of v$.address.houseNumber.$errors" :key="index">
+						<small class="p-error">{{ error.$message }}</small>
+					</span>
+				</span>
+				<small
+					v-else-if="
+						(v$.address.houseNumber.$invalid && submitted) || v$.address.houseNumber.$pending.$response
+					"
+					class="p-error">
+					{{ v$.address.houseNumber.required.$message.replace("Value", "Value") }}
+				</small>
+			</div>
+			<div class="p-float-label w-7 pl-1">
+				<InputNumber id="localNumber" v-model="v$.address.localNumber.$model" />
+				<label
+					for="localNumber"
+					:style="[v$.address.houseNumber.$invalid && submitted ? { top: '34%' } : { top: '50%' }]">
```

Comment:
I think this line isn't needed, since it only breaks the label:
![image](https://user-images.githubusercontent.com/51292024/160253723-7de0adf1-6da0-4bc2-b9a9-482ffc76686b.png)

Reply:
Removed this line but now it looks odd when house number field is invalid and I don't know how to fix it so that it looks good both when fields are valid and invalid.